android/src/main/java/ie/gov/tracing/Tracing.kt

```diff
@@ -115,8 +115,8 @@ class Tracing {
         private const val STATUS_STOPPING = "STOPPING"
 
         var status = STATUS_STOPPED
-        private var exposureStatus = EXPOSURE_STATUS_UNKNOWN
-        private var exposureDisabledReason = ""
+        private var exposureStatus = EXPOSURE_STATUS_UNAVAILABLE
+        private var exposureDisabledReason = "initialising"
```

Comment:
minor note: need to determine if the app is going to use American english spelling or Everywhere else english spelling ;-)

Reply:
Renamed to `starting` to avoid this issue